perun-utils/systemd/perun-ldapc.service

```diff
@@ -0,0 +1,19 @@
+# Systemd unit file for perun-ldapc
```

Comment:
This file seems to be unnecessary. The first invocation of "systemctl start perun-ldapc" generates file /run/systemd/generator.late/perun-ldapc.service using systemd-sysv-generator

Reply:
Does it work like this on RHEL too ? I'm not sure it works like this on RHEL, since they ditched init.d scripts completely and I installed tomcat and engine systemd services manually.